### PR TITLE
fixed typos and changed some words

### PR DESCRIPTION
**Pull request by astroidex**
* see ticket https://github.com/mapbender/mapbender/issues/495
* fixed some typos and used button instead of word tabs in sidepane

### CODE REVIEW

File: src/Mapbender/ManagerBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -71,9 +71,9 @@ mb:
       zoombar:
         draggable: Draggable
       filter:
-        source: 'Filter source'
-        layerset: 'Filter layerset'
-        application: 'Filter application'
+        source: 'Filter'
+        layerset: 'Filter'
+        application: 'Filter'
```
Comment (AxxL) [inline src/Mapbender/ManagerBundle/Resources/translations/messages.en.yml]: Nein. Design passt nicht.

File: src/Mapbender/ManagerBundle/Resources/translations/messages.en.yml [hunk 2]
```diff
@@ -91,9 +91,7 @@ mb:
         add: 'Add instance'
         show_hide: 'Toggle show/hide instance'
         edit: 'Edit instance'
-        delete:
-          title: 'delete instance'
-          confirm: 'Do you really want to remove the instance with the title <span class="italic">%instance_title%</span>?'
+        delete: 'Delete instance'
```
Comment (AxxL) [inline src/Mapbender/ManagerBundle/Resources/translations/messages.en.yml]: Nein. Komplette Änderung

**Comment by eSlider** [comment]
@astroidex, we can't merge this one, course of structe change and simplifing titles

File: src/Mapbender/ManagerBundle/Resources/translations/messages.en.yml (hunk 1 above)
Comment (werrolf) [reply src/Mapbender/ManagerBundle/Resources/translations/messages.en.yml]: Sehe kein Problem, das entspricht der deutschen Version. Da steht auch nur "Filter".

**Comment by werrolf** [comment]
Capitalization fix has been committed to hotfix/translations, other changes discarded.

**Pull request closed without merging**